Apps/Playground/Android/app/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -126,7 +131,11 @@ extern "C"
         {
             ANativeWindow *window = ANativeWindow_fromSurface(env, surface);
             g_runtime->Dispatch([window, width = static_cast<size_t>(width), height = static_cast<size_t>(height)](auto env) {
-                g_graphics->UpdateWindow<void*>(window);
+                GraphicsConfiguration graphicsConfig = GraphicsConfiguration();
```

Comment:
I figured using the default constructor and  populating the fields independently in our sample playground gives us the flexibility to expand the `GraphicsConfiguration` object without needing to update existing code that uses this struct. I considered using a list initializer as well, however we're still constrained by the ordering of the members in the struct.

Reply:
The suggestion to have a constructor take arguments isn't really more than a nit, so it's not that important. That said, you can still choose to not use the constructor if it doesn't fit your use case, and you can have multiple constructors if you have several typical use cases. In this case, since you're writing the same four or five lines of code multiple times in multiple places, this initialization scenario seems common enough to warrant a constructor (which, if the struct is later expanded with other members, could just leave those blank if it wants to since they by definition won't pertain to this usage). As I said, though, it's a nit, so just a judgement call. The main thing this comment was to document was the same thing as the long one: the initialization code.